src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      data-qa="big"
+      action="#"
+      method="get"
+      class="form-1"
+    >
+      <a href="#" class="image-1">
+        <img src="./images/Search.svg" alt="search_image">
+      </a>
+      <input 
+        data-qa="keypress"
+        class="search-1"
```

Comment:
```suggestion
        class="big-search"
```
Do not use integers in the names of yours classes

Reply:
Fix it everywhere